### PR TITLE
Site: Updates text on /measure.

### PR DESCRIPTION
**Pull request by meggynw**
For rationale, see go/web.dev-uxwaudit-measure (Googlers-only link)

### CODE REVIEW

**Comment by meggynw** [comment]
@robdodson @Meggin @kusler @harleenkbatra08 for review

**Comment by robdodson** [comment]
Sorry for the lag, I'll review this today

**Comment by robdodson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: content/measure.html [hunk 1]
```diff
@@ -70,13 +70,20 @@ <h2>Learn how to fix it</h2>
...
-        Lighthouse is an open-source, automated tool for improving the quality of your web apps.
-        It is integrated directly into the Chrome DevTools Audits panel, but can also be
-        run from the command line, as a <a href="https://www.npmjs.com/package/lighthouse" class="gc-analytics-event" data-label="measure, Lighthouse module lnk" data-action="click">Node module</a>,
-        or by installing the <a href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk" class="gc-analytics-event" data-label="measure, Lighthouse crx link" data-action="click">Chrome Extension</a>.
-        Lighthouse also provides a REST API which is consumed by sites like web.dev and <a href="https://developers.google.com/speed/docs/insights/v5/get-started" class="gc-analytics-event" data-label="measure, PSI link" data-action="click">PageSpeed Insights</a>.
+        When you measure your site, web.dev uses the <a href="https://developers.google.com/web/tools/lighthouse/" class="gc-analytics-event" data-category="web.dev" data-label="measure, Lighthouse link" data-action="click">Lighthouse</a> tool. Tests are run using Lighthouse's default settings:
```
[the remark this answers is not in the document]
Comment (meggynw) [reply content/measure.html]: I was actually hoping to invalidate the need for a FAQ! 😅  Would it be easier to fix the bullet points if we make this section left-aligned?

File: content/measure.html [hunk 2]
```diff
@@ -41,26 +41,26 @@ <h1 class="hero-title no-link">Measure</h1>
...
             </div>
             <p>
-              Provide any URL, and web.dev will run a series of audits using <a href="https://developers.google.com/web/tools/lighthouse/" class="gc-analytics-event" data-category="web.dev" data-label="measure, Lighthouse link" data-action="click">Lighthouse</a>. We'll generate a report of how well you did.
+              Enter your site's URL to see how well it performs across all audits
             </p>
           </div>
           <div class="profile-step">
             <div class="numbered-header numbered-header--vertical">
-              <h2>Analyze for improvements</h2>
+              <h2>Choose what matters</h2>
```
[the remark this answers is not in the document]
Comment (meggynw) [reply content/measure.html]: Hmm, I was thinking of this as more of a user-driven choice/prioritization of "what matters," not literally configuring/choosing options about what matters. WDYT of changing to "Decide what matters"? Does that feel more accurate? If not, happy to rewrite heading + body to be more accurate!

[the remark this answers is not in the document]
Comment (robdodson) [reply content/measure.html]: Yeah I think rephrasing so there's no implication that we're talking about configuration might be good.

File: content/measure.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (robdodson) [reply content/measure.html]: Yeah I think I can rework this a bit.

**Comment by robdodson** [comment]
I'll send an update to this pull request to sort out the list issue we discussed.

File: content/measure.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (meggynw) [reply content/measure.html]: Sorry for the delayed response! I waffled on some verbs for a while, but I think something simple & straightforward like "Look at what matters" best conveys the user-initiated, non-configured action of skimming through the results. Will submit an edit in the PR :)

**Pull request closed without merging**